packages/tasks/src/local-apps.ts

```diff
@@ -54,7 +55,7 @@ brew install llama.cpp
 # Load and run the model
 llama \\
 	--hf-repo "${model.id}" \\
-	--hf-file file.gguf \\
+	--hf-file {{GGUF_FILE}} \\
```

Comment:
Nice, thanks! 

Do you have a screenshot of how this looks with multiple GGUF files?

Reply:
yes, 

with multiple files: 
<img width="894" alt="Capture d’écran 2024-06-10 à 12 12 25" src="https://github.com/huggingface/huggingface.js/assets/468620/2db0f155-680c-4ac0-ba06-50cd53b1b48b">

with a single file: